examples/derive_bits/derive-bits-ecdh.js

```diff
@@ -1,55 +1,93 @@
 import { crypto } from "k6/x/webcrypto";
 
 export default async function () {
-  try {
-    // Generate a key pair for Alice
-    const aliceKeyPair = await crypto.subtle.generateKey(
+  // Generate a key pair for Alice
+  crypto.subtle
+    .generateKey(
```

Comment:
Is there particular reason for removing `await` + `try/catch` through out the examples?

Reply:
Well, it started with https://github.com/grafana/xk6-webcrypto/pull/67#discussion_r1561013838 :sweat_smile: 

It seems like having the `then` chains better catch errors for the precise operations, doesn't it?

Reply:
Aaah ... sorry, I should've been more explicit :sweat: 

In that comment I meant that we don't do anything with the exception ... so catching it just to log it seems bad instead of letting it propagate. Also makes the example more indented without any other benefit.

But movign to `then` from `await` seems to go in the exact opposite direction.

I guess we can really drop the `try`/`catch` if they are only used for this, but moving away from `await` doesn't seem to improve readability in my case

Reply:
Personally, the `then` syntax seems better to me, but I don't really have a strong opinion here :shrug: 

I see the benefits of unifying since it's useful for the https://github.com/grafana/xk6-webcrypto/pull/77/commits/806d97e8652554760d2f0a3cd17338c41e95b000 where we check for the key words `got unexpected error,` but on the other side, we could stick with the something standard.

Reply:
Reverted with the latest force push